web/src/routes/execution.tsx

```diff
@@ -27,46 +36,70 @@ function TanCard({ tan }: { tan: string }): ReactElement {
 }
 
 function PlayerStatus({ player }: { player: Player }): ReactElement {
+  const submit = useSubmit()
   return (
     <tr>
       <td>{player.tan}</td>
-      <td>{player.status}</td>
+      <td>
+        <CsrfForm
+          onChange={(event) => submit(event.currentTarget)}
+          method="POST"
+        >
+          <input type="hidden" name="id" value="player-status" />
+          <input type="hidden" name="tan" value={player.tan} />
+          <Button
+            id={"toggle-status-" + player.tan}
+            name="alerted"
+            type="submit"
+            value={Number(player.alerted)}
+            title="Zum Ändern klicken"
+          >
+            {player.alerted ? "Alarmiert" : "Bereit"}
+          </Button>
+        </CsrfForm>
+      </td>
       <td>{player.name}</td>
-      <td>{player.action}</td>
     </tr>
   )
 }
 
-function ToggleExecution({ execution }: { execution: ExecutionData }): ReactElement {
-  const executionActive = execution.status !== ""
+function ToggleExecution({
+  execution,
+}: {
+  execution: ExecutionData
+}): ReactElement {
+  const executionActive = execution.status === ExecutionStatusEnum.enum.RUNNING
   return (
     <div className="mt-5">
-      <Form method="POST">
-        <CsrfInput />
+      <CsrfForm method="POST">
+        <input type="hidden" name="id" value="toggle-execution" />
         <Button
           name="toggle"
           value={executionActive ? "stop" : "start"}
           type="submit"
         >
           {executionActive ? "Beenden" : "Starten"}
         </Button>
-      </Form>
+      </CsrfForm>
     </div>
   )
 }
 
 export default function Execution(): ReactElement {
   const loaderData = useLoaderData()
-  const actionData = useActionData()
+
   useEffect(() => {
-    if (isExecutionData(actionData)) {
-      setExecution(actionData)
+    if (isExecutionData(loaderData)) {
+      setExecution(loaderData)
```

Comment:
I would prefer to locate this code below the definition of `execution` and `setExecution`.